L10/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Hobbit create Function</title>
+</head>
+<body>
+	<h2>Task L10</h2>
+	<div>
+		<p> Create a function that will create hobbits. That is, return an object that we will consider a hobbit. </p>
+		<p> Hobbit - object. which has fields. Name, age and counter.</p>
+		<p> The name must be passed to the function. If no name is given DO NOT create a hobbit. </p>
+		<p> Age (integer) should be generated automatically and randomly in the range from 12 to 100. We will assume that years.</p>
+		<p> The counter is also generated automatically. And it carries a meaning - how the hobbit was created in a row.</p>
+		<p> Then use this function to create 24 hobbits with different names. Output to the console as an array. </p>
+		<p> P.S. To complete the task, of course, create an html page. It is desirable to connect scripts through the "src" attribute. </p>
+	</div>
+	<!-- <h2> Shire </h2> -->
+	<!-- <div id='hobbitonia'></div> -->
+</body>
+<script src="helper.js"></script>
+<script type="text/javascript">
+
+	var hobbitArr = [];
```

Comment:
I think this array might be useless in case the "genHobitonia" will returns an new array with hobbits. 
`var hobitonia = genHobitonia();`

or even 
`console.log('result', genHobitonia());`

Reply:
0\